test/integration/runner_task_test.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+require 'kubernetes-deploy/runner_task'
+class RunnerTaskTest < KubernetesDeploy::IntegrationTest
+  def test_works
+    deploy_fixtures("hello-cloud")
```

Comment:
Please add an assertion that the template exists to `FixtureSetAssertions::HelloCloud#assert_all_up` and then call that method here.